linux_os/guide/services/nfs_and_rpc/disabling_nfs/disabling_nfs_services/group.yml

```diff
@@ -8,5 +8,3 @@ description: |-
     All of these daemons run with elevated privileges, and many listen for network
     connections. If they are not needed, they should be disabled to improve system
     security posture.
-
-platform: machine
```

Comment:
Containers have services too. Reverting this change.

Reply:
ditto (see `service_cockpit_disabled`)